content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Updating configuration via a ConfigMap
+content_type: tutorial
+weight: 20
+---
+
+<!-- overview -->
+This page provides a step-by-step example of updating configuration within a Pod via a ConfigMap and builds upon the [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) task. At the end of this tutorial, you will understand how to change the configuration for a running application. 
+This tutorial uses the NGINX image as an example.
```

Comment:
Why NGINX? We could use something else, because we're not using that image for its usual purpose of being a webserver.

You could change the image to Alpine Linux or Debian and I think it would still work OK. Is that right?

---

Extra credit: add a final step that runs _two_ containers, one writing a file in HTML format to an `emptyDir`, and the other serving the HTML via HTTP. Change the HTML based on a ConfigMap.

Reply:
I agree. In the latest commit, I have modified the examples to use the ```alpine``` image.

Reply:
I'll work on the final step of a Pod with two containers, rendering HTML on the basis of a ConfigMap.

Reply:
In the latest commit, I have added a third section that demonstrates the scenario with two containers.